MariaDB/server
# MDEV-33837: Workaround chown warnings

- [x] *The Jira issue number for this PR is: MDEV-33837*

## Description
Blindly recursive chown is not way to do it. This Workaround is not much better than just chown -R but there is small  adjustment just chown MariaDB statedir and logdir then with find only chown those files that are not correctly owned.

Fixes lintian warnings:
 * W: mariadb-server: recursive-privilege-change "chown -R" [postinst:*]
 * W: mariadb-server: recursive-privilege-change "chown -R" [postinst:*]

## Release Notes
Should not change anything


## How can this PR be tested?
Build deb packages with `debian/autobake-debs.sh`  and check if `/var/lib/mysql` check if get suid to root as it would without this commit

## Basing the PR against the correct MariaDB version

## PR quality check
